### PR TITLE
Style about

### PR DESCRIPTION
**Pull request by jaredkhan**
Implements a simple about section with some handsome faces

screen shot 2017-08-23 at 23 29 08

### CODE REVIEW

File: static/src/styles/landing/LandingAbout.styl [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+  letter-spacing: 0.15em;
+  font-weight: 500;
+  Text__headerScale(0);
+  color: $Color__mediumGrayText;
+}
+
+.LandingAbout__headline {
+  Text__headerScale(2);
+  font-family: $Text__monospacedFont;
+  text-transform: uppercase;
```
Comment (Pinpickle) [inline static/src/styles/landing/LandingAbout.styl]: Do we want this to be uppercase? The monospace font looks especially blocky in all caps.

**Comment by Pinpickle** [comment]
Looking good! :+1: 

One minor comment but otherwise happy for this to go out.

File: static/src/styles/landing/LandingAbout.styl (hunk 1 above)
Comment (jaredkhan) [reply static/src/styles/landing/LandingAbout.styl]: I'm not sure about this either, I was just going by the design for now. Lowercase also didn't look great though. Will play around

File: static/src/styles/landing/LandingAbout.styl [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+.LandingAbout__image {
+  width: 100%;
+}
+
+.LandingAbout__text {
+  flex: 1;
+}
+
+.LandingAbout__subtitle {
+  // TODO: Remove this when Text--sectinoHeading is merged in
```
Comment (Pinpickle) [inline static/src/styles/landing/LandingAbout.styl]: Is this still supposed to be here?

**Pull request merged**